src/Picasso_UniformMesh.hpp

```diff
@@ -183,15 +189,51 @@ class UniformMesh
     // Get the local grid.
     std::shared_ptr<local_grid> localGrid() const { return _local_grid; }
 
+    //// Get the mesh node coordinates.
+    // std::shared_ptr<node_array> nodes() const { return _nodes; }
+
     // Get the cell size.
     double cellSize() const
     {
         return _local_grid->globalGrid().globalMesh().cellSize( 0 );
     }
 
+  public:
+    // Build the mesh nodes.
+    template <class ExecutionSpace>
+    std::shared_ptr<node_array> nodes( const double cell_size,
```

Comment:
Why not have a `buildNodes()` function and then just have this function return the `shared_ptr` to the array that is stored by the class in the same way as `AdapativeMesh`?

Reply:
done